### ISSUE TITLE
Clarify incentives to run relays in FAQ

### ISSUE THREAD

**Issue by clemsos** [nostr-protocol/nostr#98]
The part of the FAQ about incentives to run relays is not really clear, and currently sounds a bit rhetorical. Maybe it could be clarified by listing a few of the motivations that people have to run P2P nodes ?

> What incentive is there for people to run relays?
>
> The question is misleading. It assumes that relays are free dumb pipes that exist such that people can move data around through them. In this case yes, the incentives would not exist. This in fact could be said of DHT nodes in all other p2p network stacks: what incentive is there for people to run DHT nodes?

**Comment by fiatjaf** [comment]
I agree it must be improved and many other questions must be added to the FAQ.
